### PR TITLE
Create hooks

### PR DESCRIPTION
**Pull request by nicholasdalhaug**

### CODE REVIEW

File: frontend/src/utils/hooks.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { useEffect, useRef } from "react"
+
+export const useNotOnMountEffect = (f: () => void, deps: any[]) => {
```
Comment (nicholasdalhaug) [inline frontend/src/utils/hooks.ts:3]: `useEffectNotOnMount`

File: frontend/src/utils/hooks.test.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            <Button
+                onClick={() => {
+                    setState(oldState => oldState + 1)
+                }}
+            >
+                Click me effect
+            </Button>
+            <Button
+                onClick={() => {
+                    setStateNotConnected(oldState => oldState + 1)
```
Comment (nicholasdalhaug) [inline frontend/src/utils/hooks.test.tsx:32]: Do not use oldState twice here, use different name.

Comment (nicholasdalhaug) [reply frontend/src/utils/hooks.test.tsx:32]: `renderState`

File: frontend/src/components/Action/ActionForm.tsx [hunk 3]
```diff
@@ -11,6 +11,7 @@ import { barrierToString } from '../../utils/EnumToString'
 import { DataToCreateAction } from '../../api/mutations'
 import styled from 'styled-components'
 import ActionNotesList from './ActionNotesList'
+import { useNotOnMountEffect } from '../../utils/hooks'
```
Comment (nicholasdalhaug) [inline frontend/src/components/Action/ActionForm.tsx:14]: Remove unused imports!

File: frontend/src/api/mutations.ts [hunk 4]
```diff
@@ -60,6 +46,18 @@ export const useCreateActionMutation = (): CreateActionMutationProps => {
         ${ACTION_FIELDS_FRAGMENT}
     `
 
+    const QUESTION_ACTIONS_FRAGMENT = gql`
+    fragment QuestionActions on Question {
```
Comment (nicholasdalhaug) [inline frontend/src/api/mutations.ts:50]: Indent?

File: frontend/src/components/Action/QuestionActionsList.tsx [hunk 5]
```diff
@@ -48,27 +48,33 @@ const QuestionActionsList = ({ question, participants, onActionCreate, onActionE
...
-                                <Box display="flex" alignItems="center">
-                                    <Typography link onClick={() => editAction(action)}>
-                                        {action.title}
-                                    </Typography>
-                                    {action.completed && (
-                                        <Typography bold italic>
-                                            &nbsp;{'- Completed'}
+                {actions
+                    .sort((a1, a2) => {
+                        if (a1.createDate < a2.createDate) return -1
```
Comment (nicholasdalhaug) [inline frontend/src/components/Action/QuestionActionsList.tsx:53]: Use `{}`

**Review by denektenina — APPROVED** [review]

**Pull request merged**